omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx.conf.erb

```diff
@@ -191,31 +191,6 @@ http {
     }
   }
 
-  # internal service access for oc_bifrost
```

Comment:
is the other internal_lb block being used by something?

Reply:
The addons use it perhaps.

Reply:
There is no impact on chef-manage or pedant tests If I remove the chef-server api server bolck, It would be good if we create another ticket for full investigation.